### PR TITLE
workaround deprecated collections.Mapping import

### LINKED ISSUE #25: policy.py: Deprecation Warning
Run of testsuite showed to me following warning:

```
starttls_policy_cli/policy.py:220
  /home/user/starttls-policy-cli/starttls_policy_cli/policy.py:220: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    class Config(MergableConfig, collections.Mapping):
```

This issue should be addressed ASAP because Python 3.8.0 has already reached alpha 2 release.

### PR DESCRIPTION
**Pull request by Snawoot**
Fixes #25. ~Probably this PR will have linting errors, but they can be ignored because they already fixed in PR #24~

### CODE REVIEW

**Comment by Snawoot** [comment]
Cherry-picked commit with linter fixes from PR #24 in order to make tests pass. Anyway it won't disrupt mergeability.

File: starttls_policy_cli/policy.py [hunk 1]
```diff
@@ -8,6 +7,11 @@
 from starttls_policy_cli import util
 from starttls_policy_cli import constants
 
+try:
```
Comment (sydneyli) [inline starttls_policy_cli/policy.py:10]:
```suggestion
try:
    # Python 3+
```

**Review by sydneyli — COMMENTED** [review]
Thanks @Snawoot ! We should move off of Python 2.7 [soon](https://pythonclock.org/) anyways, but for now this is fine. Just added a note to remind myself which one is for Python 3+.

**Comment by Snawoot** [comment]
Changed comment a little: `collections.abc` was introduced in Python 3.3, so it may be useful for some older Python 3 versions too

**Review by sydneyli — APPROVED** [review]

**Pull request merged**